README.md

```diff
@@ -1,10 +1,204 @@
+Planetcantile provides TileMatrixSets (TMS) for planetary bodies throughout the solar system, enabling tiled map visualization and analysis for the Moon, Mars, and many other celestial bodies using standard web mapping techniques.
 
-Integrates with [morecantile](https://github.com/developmentseed/morecantile), ideally should allow downstream projects to benefit from additional TMSs defined here.
+A TileMatrixSet defines how a spatial area (like a planet) is divided into a hierarchy of tiles at different zoom levels. By implementing the [OGC TileMatrixSet standard](https://docs.ogc.org/is/17-083r4/17-083r4.html) for planetary bodies, Planetcantile makes it possible to use Earth-focused web mapping tools with data from across the solar system.
 
-Definitions go into the generate.py script to have fine grain control, alternatively json files can just be added directly to the data directory.
+## Table of Contents
+- [Features](#features)
+- [Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [Usage](#usage)
+  - [Web API](#option-1-using-the-web-api)
+  - [Python Library](#option-2-as-a-python-library)
+  - [Environment Variables](#environment-variables)
+  - [Extending with Custom Definitions](#extending-with-custom-definitions)
+- [Supported Planetary Bodies](#supported-planetary-bodies)
+- [Integration with Other Libraries](#integration-with-other-libraries)
+- [Contributing](#contributing)
+- [License](#license)
+
+## Features
+
+- <strong>Extensive Coverage:</strong> Support for ~100 celestial bodies including planets, moons, asteroids, and comets
+
+- <strong>Multiple Projection Types:</strong>
+  - Geographic (regular lat/lon) - Standard equirectangular/plate carrée projection
+  - Equidistant Cylindrical - Cylindrical projection preserving distances along meridians
+  - Web Mercator - Modified Mercator projection commonly used in web mapping
+  - Mercator - Classic Mercator projection
+  - North Polar Stereographic - Projection centered on the North Pole
+  - South Polar Stereographic - Projection centered on the South Pole
+
+
+- <strong>Coordinate System Support:</strong> Handles different coordinate systems for planetary bodies:
+  - Sphere (spherical approximation)
+  - Ocentric (planetocentric, center-based coordinates)
+  - Ographic (planetographic, surface-based coordinates)
+  
+  Multiple coordinate system options are available for many celestial bodies.
+
+- <strong>Coalesced Grids:</strong> Special grid layouts that reduce the number of tiles needed near poles, optimizing storage and performance.
+
+- <strong>Cloud Optimized GeoTIFF (COG) Support:</strong> Works with COG, STAC, and MosaicJSON through TiTiler integration
+
+- <strong>Seamless Integration:</strong> Works with morecantile, TiTiler, and other OGC-compatible web mapping tools
+
+## Getting Started
+
+### Prerequisites
+
+- Python 3.10 or higher
+- pip or conda package manager
```

Comment:
planetcantile I don't think is available with conda-forge yet, I do plan to add it there at some point though